pkg/services/dashboards/manager/folder_service_test.go

```diff
@@ -138,6 +138,14 @@ func TestFolderService(t *testing.T) {
 				require.Equal(t, f, actualFolder)
 			})
 
+			t.Run("When creating folder should return error if uid is general", func(t *testing.T) {
+				dash := models.NewDashboardFolder("Test-Folder")
+				dash.Id = rand.Int63()
```

Comment:
nit: could skip these lines for this test and just pass in dashboard title as a string. But this is super minor.

Reply:
since this is the only nit in this review, I'm skipping fixing it, but I'll take note of it for when we'll probably pass through this code base again

Reply:
Makes sense 👍 Looking forward to this being merged 😄